agent/src/beerocks/slave/son_slave_thread.cpp

```diff
@@ -4513,11 +4513,8 @@ bool slave_thread::handle_autoconfiguration_wsc(Socket *sd, ieee1905_1::CmduMess
                                                         authtype, enctype))
             return false;
 
-        std::string manufacturer =
-            std::string(tlvWscM2->manufacturer(), tlvWscM2->manufacturer_length());
-
         LOG(DEBUG) << "Controller configuration (WSC M2 Encrypted Settings)" << std::endl
-                   << "     Manufacturer: " << manufacturer << std::endl
+                   << "     Manufacturer: " << tlvWscM2->manufacturer_str() << std::endl
```

Comment:
It's really not nice to mix this change together with the big Autogenerated stuff in a single commit. i.e., please make a separate commit of this.

Reply:
ok, but this requires force pushing. So I will also rebase in the process.